.github/workflows/publish-containers.yml

```
@@ -4,7 +4,7 @@ on:
   push:
     tags:
       # Matches stable and pre-releases
-      - "[0-9]+.[0-9]+.[0-9]+*"
+      - "v?[0-9]+.[0-9]+.[0-9]+*"
```

I suggest only expecting one format so we don't need to support two different tag formats:

```suggestion
      - "v[0-9]+.[0-9]+.[0-9]+*"
```